webservices/tasks/download.py

```diff
@@ -119,8 +119,9 @@ def get_s3_name(path, qs):
     # TODO: consider base64 vs hash
     raw = '{}{}'.format(path, qs)
     hashed = hashlib.sha224(raw.encode('utf-8')).hexdigest()
-    return '{}.csv'.format(hashed)
-
+    prefix = 'user_downloads/'
```

Comment:
Can we make the prefix `user-downloads` instead of `user_downloads` because we're using `bulk-downloads`?

✅ Verified files exist with `aws s3api list-objects-v2 --max-items 10 --bucket "<bucket-name>" --prefix=user_downloads`

Reply:
If you agree, could you also squash these two commits?

Reply:
Thanks @lbeaufort, I'm all for consistency. Commits have been squashed into 1 as well.

Reply:
Thanks @patphongs!